### DISCUSSION TITLE
related with career

### DISCUSSION THREAD

**Discussion by Kirtishukla2004** [community/community#32253] [Other Feature Feedback, Questions, & Ideas]
is college or university matters to get job at a very good company with a high pakage

**Comment by queenofcorgis** [comment]
Hi @shukla2004,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your additional post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

**Comment by Kirtishukla2004** [comment]
OK
SORRY
